src/fetchers/externalYieldRateFetcher.ts

```diff
@@ -9,6 +9,7 @@ import { contracts } from '../contracts';
 import { fetchValuation } from './priceFetcher';
 import { Token, TokenAmount } from '../entities';
 const axios = require('axios');
+BigNumberjs.config({POW_PRECISION: 20});
```

Comment:
Not really need to define this

Reply:
BigNumber is keeping all decimals of the number during multiplication and it will be slow. The reason for this code is to limit its precision.